src/components/TransferNotification.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <transition
+    appear
+    name="fade"
+  >
+    <div class="transfer-notification">
+      <div class="content">
+        <ae-icon
+          fill="alternative"
+          face="round"
+          name="check"
+        />
+        <div class="title">
+          Transaction completed
+          <div class="subtitle">
+            You've sent {{ amount.toString() }} AE
+          </div>
+        </div>
+      </div>
+
+      <div class="footer">
+        <ae-button
+          :to="`https://explorer.aepps.com/#/tx/${tx}`"
+          fill="neutral"
+          plain
+        >
+          View on explorer
+        </ae-button>
+        <ae-button
+          fill="neutral"
+          plain
+          @click="copyTx"
+        >
+          Copy tx hash
+        </ae-button>
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import BigNumber from 'bignumber.js';
+import copy from 'clipboard-copy';
+import { AeIcon } from '@aeternity/aepp-components-3';
+import AeButton from './AeButton.vue';
+
+export default {
+  components: {
+    AeIcon,
+    AeButton,
+  },
+  props: {
+    tx: {
+      type: String,
+      default: '',
+    },
+    amount: {
+      type: BigNumber,
+      default: BigNumber(0),
+    },
+  },
+  data: () => ({
+    migratedBalance: '',
+  }),
+  methods: {
+    copyTx() {
+      copy(this.tx);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '~@aeternity/aepp-components-3/src/styles/placeholders/typography.scss';
+@import '~@aeternity/aepp-components-3/src/styles/variables/colors.scss';
+
+.transfer-notification {
+  &.fade-enter-active, &.fade-leave-active {
+    transition: opacity 0.25s ease-out;
+  }
+
+  &.fade-enter, &.fade-leave-to {
+    opacity: 0;
+  }
+
+  &.fade-enter-to, &.fade-leave {
+    opacity: 1;
```

Comment:
Default styles can be omitted in this case. Also, should we define styles for `leave` state?